draft-ietf-mimi-arch.md

```diff
@@ -531,6 +531,29 @@ senders to authenticate the origin of the message.
 
 For a deeper discussion of identity, see {{?I-D.mahy-mimi-identity}}.
 
+# Minimal Metadata Mode
+
+In MIMI's regular mode of operation, the identity of room participants is
+visible to the room's Hub. For cases where this is not desirable, MIMI supports
+a Minimal Metadata mode, where per-group pseudonyms take the place of user- and
```

Comment:
> per-group pseudonyms

It would be nice to clarify that if you have a set of groups where you want correlation (my knitting friends), users/clients can share a pseudonym across those groups. Pseudonyms must not be shared across groups unless correlation amonth those groups is desired. (The default treatment would be to use a separate pseudonym per group).